distribution/docs/src/main/resources/content/_managing/_running/migrate-command.adoc

```diff
@@ -9,7 +9,7 @@
 
 [NOTE]
 ====
-Performing a data migration will may create, update, or delete existing metacards within the system.
+Performing a data migration will create, update, or delete existing metacards within the system.
```

Comment:
Keep it present tense:
"Performing a data migration creates, updates, or deletes existing metacards within the system."